src/api/jet/utilities.ts

```diff
@@ -10,6 +10,9 @@ export async function query<T>(
     method: "POST",
     headers: {
       "Content-Type": "application/json",
+      "Authorization": `Basic ${
+        config.authentications[config.jetEndpoint].token
```

Comment:
用户可能没有设置，或者这台服务器不需要 basic authentication

```suggestion
        config.authentications?[config.jetEndpoint]?.token
```